### PR TITLE
Add room modal fixes #13

### LINKED ISSUE #13: Modal to add a page
Create button and modal to add a page on the index page

### PR DESCRIPTION
**Pull request by bkawk**
Add room button added to top left, when clicked displays model with buttons

### CODE REVIEW

**Comment by sbeleidy** [comment]
twitter-login element shouldn't be on this branch. 
It also shouldn't open up a modal. It should automatically redirect to twitter. Can we remove twitter-login from this branch and add it on another one? We can discuss its features there.

As for add-room modal, can we add a URL field as well? 

(This part isn't critical) Can we do the "add room" button as a paper-fab instead of a button in the nav at the top? We would want to follow the material design guidelines for that. https://www.google.com/design/spec/components/buttons-floating-action-button.html#buttons-floating-action-button-floating-action-button

**Comment by bkawk** [comment]
The URL and the FAB have been added

File: app/index.html [hunk 1]
```diff
@@ -106,9 +106,9 @@
 
           <span class="space"></span>
 
-          <!-- Toolbar icons -->
-          <paper-icon-button icon="refresh"></paper-icon-button>
-          <paper-icon-button icon="search"></paper-icon-button>
+          <!-- Toolbar icons -->    
+          <room-add>11</room-add>
```
Comment (sbeleidy) [inline app/index.html:110]: why do you have "11" in there?

**Comment by sbeleidy** [comment]
- [x] Can you also remove the added blank lines?
- [x] What do you think about renaming the modal to "Create convoo"?
- [x] Can we make the modal a bit bigger?
- [ ] I'm wondering if using the dialog is the right way to go (https://www.google.com/design/spec/components/dialogs.html#dialogs-specs). What do you think about making this just go to another page? 

Let me know your thoughts on this. I'm fine with merging what we have now but addressing these things later as well. That way we can potentially start connecting the backend and work on this list at the same time.

**Comment by sbeleidy** [comment]
Hey, what's next for this?

**Comment by bkawk** [comment]
We can move this to a new page once we have the main layout in place, there is no need for the modal as all the pages are cached, we almost have the custom elements done, image resizer and the tags

**Comment by sbeleidy** [comment]
Should we use this branch then or start from the new master?

**Comment by bkawk** [comment]
Lets kill this branch

**Pull request closed without merging**